Podcast/InternalProfileViewController.swift

```diff
@@ -45,11 +45,22 @@ class InternalProfileViewController: UIViewController, UITableViewDelegate, UITa
         view.addSubview(tableView)
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+//        navigationItem.backBarButtonItem = nil
+    }
+    
     // MARK: InternalProfileHeaderViewDelegate
     
     func internalProfileHeaderViewDidPressViewProfile(internalProfileHeaderView: InternalProfileHeaderView) {
         let myProfileViewController = ExternalProfileViewController()
-        myProfileViewController.user = System.currentUser
+//        let navigationVC = UINavigationController(rootViewController: myProfileViewController)
+//        navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
+        if let currentUser = System.currentUser {
+            myProfileViewController.fetchUser(id: currentUser.id)
+//            myProfileViewController.fetchUser(id: "9d417b19-2474-11e7-be6b-3f00bd823d20") // Test for other people 
+        } else {
+            myProfileViewController.fetchUser(id: "")
```

Comment:
there's no case where you make it to this point and you don't have a current user, so do a guard let instead of an if let and just return in the else condition